admin/src/components/core/cards/dashboard/Sidenav.js

```diff
@@ -1,11 +1,26 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
+import { NavLink, withRouter } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
 import Icon from './Icon';
 import IconOverview from './../../../../assets/icons/overview.svg';
 import IconProcessors from './../../../../assets/icons/processors.svg';
 import dashLogo from './../../../../assets/images/dash-logo.svg';
 
+import session from '../../../../helpers/Session';
+
 const Sidenav = props => {
+  // TODO: move when refactoring dashboard to presentational and container components
```

Comment:
This is temporary until we do the presentational/container components refactoring